libs/packages/sam-formly/src/lib/formly-filters/sds-filters.component.ts

```diff
@@ -117,37 +118,37 @@ export class SdsFiltersComponent implements OnInit {
     }
   }
 
-  addOption() {
-    const updatedFields: FormlyFieldConfig[] = [];
-    this.fields.forEach(field => {
-      if (field) {
-        if (field.fieldGroup) {
-          field.fieldGroup.forEach(subField => {
-            if (subField.type == 'input') {
-              field.modelOptions.updateOn = 'blur';
-            } else if (subField.type == 'autocomplete') {
-              field.templateOptions.essentialModelFields = true;
-            }
-          });
-        } else {
-          if (field.type == 'input') {
-            field.modelOptions.updateOn = 'blur';
-          } else if (field.type == 'autocomplete') {
-            field.templateOptions.essentialModelFields = true;
+  checkForHide() {
+    let fieldWithValue = this.convertToParam(this.model);
+    let keys = [];
+    Object.keys(fieldWithValue).map(key => {
```

Comment:
Please add note, this is difficult to follow

Reply:
This is for getting the model which has a value.. so that I can modify the hide expression.

Reply:
Please note that in the code